src/javaRosa.js

```diff
@@ -171,7 +171,14 @@ define([
             return Boolean(this.hasForm('default') || 
                            this.hasForm('long')    || 
                            this.hasForm('short'));
-        }
+        },
+        forEachLogicExpression: function (fn) {
+            var text = this.get(),
```

Comment:
This gets the value for the default language. What do you think about iterating over all languages?